tests/unittests/test_helpers.py

```diff
@@ -34,4 +35,42 @@ def test_get_ipath_and_empty_instance_id_returns_none(self):
 
         self.assertIsNone(mypaths.get_ipath())
 
+
+class Testcloud_init_project_dir:
+    top_dir = test_helpers.get_top_level_dir()
+
+    @staticmethod
+    def _get_top_level_dir_alt_implementation():
+        """Alternative implementation for comparing against.
+
+        Note: Recursively searching for .git/ fails during build tests due to
+        .git not existing. This implementation assumes that ../../../ is the
+        relative path to the cloud-init project directory form this file.
+        """
+        out = Path(__file__).parent.parent.parent.resolve()
+        return out
+
+    def test_top_level_dir(self):
+        """Assert the location of the top project directory is correct"""
+        assert 1 == len(
```

Comment:
Nit: `assert self.top_dir == self._get_top_level_dir_alt_implementation()` should work here, right?